### DISCUSSION TITLE
Week 5

### DISCUSSION THREAD

**Discussion by audreyhki** [USU-ITLS-x265-23-summer/class-repo#131] [General]
<h2>JS</h2>

- I like that FreeCodeCamp offers a list of boxes that are ticked based on how close you are to achieving the final line of code. This breaks down the goal for you into simpler, bite-sized, steps which can help you complete the code when you find yourself stuck. 

- I dislike the instructional method on FreedCodeCamp because, unlike the html/css instruction, you are unable to see the previous code you have completed and draw back on previously learned concepts. 

- I would change the instructional approach by allowing students to go back on previous tasks they have completed and show them the code they have already completed. When returning to a previously completed task, it does not show the changes you made, it automatically resets.

<h2>HTML/CSS</h2>

- I like that you are able to view every element/attribute/value/etc that you have previously coded. It makes it easier to complete tasks following because you can use context clues from your previous code and be reminded of ways created elements prior. 

- I dislike that the instructions for html/css do not include a list of boxes that can be ticked. Instead of telling you exactly what's wrong with your code, it provides "hints". 

- I'm not sure what I would change, I've really enjoyed my time participating in the html/css lessons. It's begun clicking for me with repeated exposure and the lessons gently introduce advanced tactics. 

**_Note:_** I like that with both courses, you are able to skip a lesson. This has been super useful especially when I'm stuck. Usually, I can complete the task after the one I'm stuck on and then return to the task I wasn't able to complete, with more knowledge or a different perspective. 

<h2>Flexbot Froggy</h2>

- I liked the ability to click on commands instead of having to type them out individually each time. 

- I disliked how quickly it was over!

- I would change Flexbot Froggy by adding more levels.

<h2>Projects</h2>

I think my burger menu project idea could really benefit from the exercises in CSS building a cafe menu.

**Comment by Skye404** [comment]
I also dislike how some of the activities don't let us go back and revisit the same code. It forces you to reinput the code again. I wish it saved my work, so I could go back and view it again. That would be really beneficial.

**Comment by metasean** [comment]
I completely agree with the frustration of not retaining and continuing with the code you've already submitted. I know why, but it's still frustrating.
